src/models/vouch.py

```diff
@@ -0,0 +1,37 @@
+"""
+Script to query and display total funds distributed for specified accounting period.
+"""
+from dataclasses import dataclass
+
+from duneapi.types import Address
+
+from src.utils.dataset import index_by
+
+# pylint: disable=line-too-long
+RECOGNIZED_BONDING_POOLS = [
+    "('\\x8353713b6D2F728Ed763a04B886B16aAD2b16eBD'::bytea, 'Gnosis', '\\x6c642cafcbd9d8383250bb25f67ae409147f78b2'::bytea)",
+    "('\\x5d4020b9261F01B6f8a45db929704b0Ad6F5e9E6'::bytea, 'CoW Services', '\\x423cec87f19f0778f549846e0801ee267a917935'::bytea)",
```

Comment:
```suggestion
    "('\\x8353713b6D2F728Ed763a04B886B16aAD2b16eBD'::bytea, 'Gnosis', "
    "'\\x6c642cafcbd9d8383250bb25f67ae409147f78b2'::bytea)",
    "('\\x5d4020b9261F01B6f8a45db929704b0Ad6F5e9E6'::bytea, 'CoW Services', "
    "'\\x423cec87f19f0778f549846e0801ee267a917935'::bytea)",
```

Reply:
If you think this is still readable enough and want to remove the pylint disable.

Reply:
Sure, but since you change suggestion doesn't include removing pylint-disable, I am going to have to go and do this locally.

Reply:
Ah yuck. Sorry I forgot about that. Thanks!

Reply:
Fixed.